### ISSUE TITLE
Reminder/Notification

### ISSUE THREAD

**Issue by DeanGross** [MicrosoftDocs/entra-docs#77]
Please provide some information about How we can get a reminder sent to notify us when a secret is going to expire

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 08f516f2-8b4d-7043-387a-96c746f47598
* Version Independent ID: 6a932262-af92-8ac4-6b84-5adc0bf7e894
* Content: [Microsoft Entra recommendation - Renew expiring application credentials (preview) - Microsoft Entra ID](https://learn.microsoft.com/en-gb/entra/identity/monitoring-health/recommendation-renew-expiring-application-credential?toc=%2Fentra%2Fidentity%2Fworkload-id%2FTOC.json&bc=%2Fentra%2Fidentity%2Fworkload-id%2Fbreadcrumb%2Ftoc.json)
* Content Source: [docs/identity/monitoring-health/recommendation-renew-expiring-application-credential.md](https://github.com/MicrosoftDocs/entra-docs/blob/main/docs/identity/monitoring-health/recommendation-renew-expiring-application-credential.md)
* Service: **active-directory**
* Sub-service: **report-monitor**
* GitHub Login: @shlipsey3
* Microsoft Alias: **sarahlipsey**

**Comment by SaibabaBalapur-MSFT** [comment]
@DeanGross 
Thanks for your feedback! We will investigate and update as appropriate.

**Comment by SaibabaBalapur-MSFT** [comment]
@DeanGross You can get a reminder sent to notify you when a secret is going to expire by using Azure AD's certificate management feature. Azure AD will send an email notification 60, 30, and 7 days before the SAML certificate expires. You may add more than one email address to receive notifications. To specify the emails you want the notifications to be sent to, see [Add email notification addresses for certificate expiration](https://docs.microsoft.com/en-us/azure/active-directory/manage-apps/manage-certificates-for-federated-single-sign-on#add-email-notification-addresses-for-certificate-expiration).

**Comment by DeanGross** [comment]
Thanks but what about secrets that are not from Certificates?

**Comment by SaibabaBalapur-MSFT** [comment]
@DeanGross For secrets that are not from certificates, it is recommended to use Azure Key Vault to store and manage them. You can use managed identities for Azure resources or Azure AD App authentication to access the secrets stored in Azure Key Vault.

It talks about renewing expiring application credentials. The article recommends using Azure AD App authentication to access the credentials stored in Azure Key Vault. 

So, in summary, it is recommended to use Azure Key Vault to store and manage secrets, and then use Azure AD App authentication or managed identities for Azure resources to access the secrets.
